release_notes/ocp-4-20-release-notes.adoc

```diff
@@ -594,6 +594,13 @@ Previously, creating an SR-IOV network required a cluster administrator to confi
 
 For more information, see xref:../networking/hardware_networks/configuring-namespaced-sriov-resources.html#configuring-namespaced-sriov-resources[Configuring namespaced SR-IOV resources].
```

Comment:
Not yours but pretty sure it should be .adoc rather than .html in xref

Reply:
That is my bad can you fix it when in here sorry @slovern should be extension .adoc